### DISCUSSION TITLE
filtering bookmarks

### DISCUSSION THREAD

**Discussion by rafael844** [sepinf-inc/IPED#1238] [Q&A]
Is there a way in cmd to filter bookmarks as in GUI report generation?
Ex: Bookmark.iped has book1, book2 and book3. 
I only want to generate a new report of book3 from Bookmark.iped.

**Comment by wladimirleite** [comment]
As far as I remember, report generation using command line includes in the report all checked items (checkbox marked). So in the case you mentioned, you would need to mark only items in book3 before running the report.

**Reply by lfcnassif** [reply]
Thanks @tc-wleite. First the cmd line will try to pick up the bookmarks checked on UI, if none is checked, it will fallback to ancient behavior of picking up the checked items. So today there is no way of specifying them in the cmd line directly.

**Comment by rafael844** [comment]
I see, I will try to create multiple bookmarks then. Thank you.
